test/elixir/test/bulk_docs_test.exs

```diff
@@ -129,6 +129,15 @@ defmodule BulkDocsTest do
     assert Enum.at(rows, 2)["error"] == "conflict"
   end
 
+  @tag :with_db
+  test "bulk docs raises transaction_too_large error for transaction larger than 10MB", ctx do
+    docs = [%{_id: "0", a: random_string(16_000_000)}]
+    set_config_raw("couchdb", "max_document_size", "67108864")
```

Comment:
Will this automatically reset if the test ends (either ok or fails)? We wouldn't want to alter the max doc size for other tests here.

Reply:
No, the setting on `[couchdb][max_document_size]` will not be reverted after one test or test suite ends. I added codes to revert them. I believe that currently we are fine with this test.